ui/src/components/Settings/Notifications/CreateNotificationModal/index.tsx

```diff
@@ -0,0 +1,302 @@
+      >
+        <div>
+          <form className="space-y-4">
+            {/* Config Name */}
+            <div className="form-row">
+              <label htmlFor="name" className="text-label">
+                Name *
+              </label>
+              <div className="form-input">
+                <div className="form-input-field">
+                  <input
+                    type="text"
+                    name="name"
+                    defaultValue={props.selected?.name}
+                    onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                      (nameRef.current = event.target.value)
+                    }
+                  ></input>
+                </div>
+              </div>
+            </div>
+            {/* Enabled */}
+            <div className="form-row">
+              <label htmlFor="enabled" className="text-label">
+                Enabled
+              </label>
+              <div className="form-input">
+                <div className="form-input-field">
+                  <input
+                    type="checkbox"
+                    name="enabled"
+                    id="enabled"
+                    defaultChecked={props.selected?.enabled}
+                    onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                      event.target.value === 'on'
+                        ? (enabledRef.current = true)
+                        : (enabledRef.current = false)
+                    }
+                  ></input>
+                </div>
+              </div>
+            </div>
+            {/* Select agent */}
+            <div className="form-row">
+              <label htmlFor="ssl" className="text-label">
+                Agent *
+              </label>
+              <div className="form-input">
+                <div className="form-input-field">
+                  <select
+                    id="agent"
+                    name="agent"
+                    onChange={(e) => {
+                      setFormValues({})
+                      setTargetAgent(availableAgents[Number(e.target.value)])
+                    }}
+                    className="rounded-l-only"
+                  >
+                    {availableAgents?.map((agent, index) => (
+                      <option
+                        key={`agent-${index}`}
+                        value={index}
+                        selected={
+                          props.selected
+                            ? agent.name === props.selected?.agent
+                            : false
+                        }
+                      >
+                        {`${agent.name}`}
+                      </option>
+                    ))}
+                  </select>
+                </div>
+              </div>
+            </div>
+
+            <div className="form-row">
+              <label htmlFor="about-scale" className="text-label">
+                Days for About To Be Handled
```

Comment:
I was a bit confused at first when I saw this as I hadn't read the types. Perhaps move this after Types and only make it appear when selecting Media About To Be Handled?

Reply:
I spent too much time considering how to name this and eventually settled on this option, though I wasn't entirely satisfied with it.

You're right that showing this only when a notification provider of that type is selected would be a better approach. But, since the types are configured in the settings, it might still be a bit confusing for some users?